site/docs/reference/Connectors/capture-connectors/sqlserver.md

```diff
@@ -0,0 +1,277 @@
+---
+sidebar_position: 6
+---
+# Microsoft SQL Server
+
+This connector uses change data capture (CDC) to continuously capture updates in a Microsoft SQL Server database into one or more Flow collections.
+
+It’s available for use in the Flow web application. For local development or open-source workflows, [`ghcr.io/estuary/source-sqlserver:dev`](https://ghcr.io/estuary/source-sqlserver:dev) provides the latest version of the connector as a Docker image. You can also follow the link in your browser to see past image versions.
+
+## Supported versions and platforms
+
+This connector supports SQL Server 2017 and later on major cloud providers,
+as well as self-hosted instances.
+Setup instructions are provided for the following platforms:
+
+* [Self-hosted SQL Server](#setup-self-hosted-sql-server)
+* [Azure SQL Database](#setup-azure-sql-database)
+* [Amazon RDS for SQL Server](#setup-amazon-rds-for-sql-server)
+* [Google Cloud SQL for SQL Server](#setup-google-cloud-sql-for-sql-server)
+
+## Prerequisites
+
+To capture change events from SQL Server tables using this connector, you need:
+
+* For each table to be captured, a primary key should be specified in the database.
+If a table doesn't have a primary key, you must manually specify a key in the associated Flow collection definition while creating the capture.
+[See detailed steps](#specifying-flow-collection-keys).
+
+* [CDC enabled](https://learn.microsoft.com/en-us/sql/relational-databases/track-changes/enable-and-disable-change-data-capture-sql-server?view=sql-server-ver16)
+on the database and the individual tables to be captured.
+(This creates *change tables* in the database, from which the connector reads.)
+
+* A **watermarks table**. The watermarks table is a small “scratch space” to which the connector occasionally writes a small amount of data to ensure accuracy when backfilling preexisting table contents.
+
+* A user role with:
```

Comment:
There's one additional permission that the capture user needs to have, which is that they must have access to the change data itself [1]. This is controlled through the `@role_name` parameter when CDC is enabled on a particular table, and the capture user needs to either be named explicitly there or be (transitively) a member of whatever role is specified.

[1] This access is distinct from the change tables `SELECT` access for some reason -- if they only have `SELECT` on the `cdc` schema we can poll for change events but the result is just a bunch of `NULL` values.